exonum/src/blockchain/builder.rs

```diff
@@ -171,13 +185,34 @@ impl BlockchainBuilder {
     }
 }
 
+/// DTO for service instance configuration.
```

Comment:
It seems to me that the abbreviation does not look good here.

Reply:
Perhaps the reason is that Rust does not have DTOs? I can suggest `Service instance instantiation parameters`

Reply:
Reworded